app/src/main/java/com/example/myapplication/vm/LoginViewModel.kt

```diff
@@ -26,6 +25,18 @@ class LoginViewModel(app: Application) : AndroidViewModel(app) {
     val invalidEmail: LiveData<Boolean>
         get() = _invalidEmail
 
+    private val _validEmail = MutableLiveData<Boolean>()
+    val validEmail: LiveData<Boolean>
+        get() = _validEmail
```

Comment:
You are storing a boolean for both valid email and invalid email. You can create a LiveData called validEmail, and set it true or false.